### ISSUE TITLE
Trying to install lc0 on linux; git comand fails

### ISSUE THREAD

**Issue by emdio** [LeelaChessZero/lc0#528]
So trying to install lc0 for linux, following the instructions in https://github.com/LeelaChessZero/lc0, after trying the git command for cloning the repo I get this error;

$ git clone -b release --recurse-submodules https://github.com/LeelaChessZero/lc0.git
Cloning into 'lc0'...
fatal: Remote branch release not found in upstream origin

**Comment by emdio** [comment]
Oh, I cloned "master" instead of "release". I assume the rest of instructions will be the same.

**Comment by ssb22** [comment]
This issue should probably be reopened as it is a bug in README.md (unless there *will* be a release branch soon?)

**Comment by mooskagh** [comment]
Yeah, we'll need to fetch lastest tag / latest stable tag instead.
Probably can be parsed from https://api.github.com/repos/LeelaChessZero/lc0/releases/latest

**Comment by borg323** [comment]
For linux, I have used `git checkout $(git tag --list |grep -v rc |tail -1)`

**Comment by borg323** [comment]
Alternatives: 
`git checkout $(git describe --match v?.??.? --abbrev=0)` (linux)
`for /f %a in ('"git describe --match v?.??.? --abbrev=0"') do git checkout %a` (windows cmd)
